fluture.js

```diff
@@ -41,7 +41,7 @@
   }
 
   function isFuture(m){
-    return (m instanceof FutureClass) || Boolean(m) && m['@@type'] === TYPEOF_FUTURE;
+    return (m instanceof Future) || Boolean(m) && m['@@type'] === TYPEOF_FUTURE;
```

Comment:
The parens to the left of the `||` are unnecessary. Also, `m != null` would be clearer than `Boolean(m)`.

What's the purpose of the `instanceof` check?